.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,82 @@
+name: Ensure downstream dependencies are not broken by this commit
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ gpavlov/dependencies ]
```

Comment:
Did you mean to hardcore this in here?

Reply:
I am expecting few more updates to the script (for example notification to slack) so I am keeping it here to enable quick experementation

Reply:
I suggest keeping it as draft until it's ready to review.